packages/metrics/tests/unit/Metrics.test.ts

```diff
@@ -76,7 +76,7 @@ describe('Class: Metrics', () => {
       metrics.addDimension(additionalDimension.name, additionalDimension.value);
       const loggedData = metrics.serializeMetrics();
 
-      expect(loggedData._aws.CloudWatchMetrics[0].Dimensions[0].length).toEqual(1);
+      expect(loggedData._aws.CloudWatchMetrics[0].Dimensions[0].length).toEqual(2);
```

Comment:
Now that there is a default service name, the `service` dimension is always set. Hence these are incremented

Reply:
Let's add this as a comment above the `expect..` line please.

Reply:
Done